ha/k8s_setup/ha_setup.py

```diff
@@ -316,14 +318,32 @@ def _add_cvg_health(self) -> None:
                                            resource_type=CLUSTER_ELEMENTS.CVG.value,
                                            resource_id=cvg)
 
-    def _add_health_event(self, node_id: str, resource_type: str, resource_id: str) -> None:
+    def _add_disk_health(self) -> None:
         """
-        Add health events for multiple resources (e.g. Node, CVG, disk)
+        Add disk health
+        """
+        _, nodes_list = self._confStoreAPI.get_cluster_cardinality()
+        for node in nodes_list:
+            cvg_list = ConftStoreSearch.get_cvg_list(self._index, node)
```

Comment:
This logic is same as in add_cvg and code till here is getting executed twice, un-necessarily. 
We can have a new function to update disks for a particular cvg only and  add this call in _add_cvg_health thereby reusing the same code and reducing duplicate execution

Reply:
Done.